kodilla-spring/src/main/java/com/kodilla/spring/calculator/Display.java

```diff
@@ -0,0 +1,15 @@
+package com.kodilla.spring.calculator;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
+
+@Component
+@Configuration
```

Comment:
what @Configuration annotation mean?

Reply:
I shouldn't have used to @Configuration with @Component. 
I could use @Configuration with @Beans in methods.